app/main.development.js

```diff
@@ -38,8 +38,48 @@ const installExtensions = async () => {
   }
 };
 
+const launchDCRd = async () => {
+  var childProcess = require('child_process');
+  var args = ['--piperx=0','--pipetx=1','--lifetimeevents']
```

Comment:
0 can't be used, it is the default and it means the feature is not enabled.  Need to make a new pipe.

I don't think you are using any of the lifetime events at this point, but those should not be going to stdout because it is cluttered by logs.  Remove it or create a new pipe for this.